src/workerd/io/compatibility-date.capnp

```diff
@@ -70,6 +65,13 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # ("FL" refers to Cloudflare's HTTP proxy stack which is used for all outbound requests. Flags
   # with this annotation have no effect when `workerd` is used outside of Cloudflare.)
 
+  annotation experimental @0xe3e5a63e76284d88 (field):Void;
```

Comment:
Let's please mark `tcpSocketsSupport`, `nodeJs18CompatExperimental`, and `obsolete14` (for the test's sake) as `$experimental`.

Reply:
Just a clarification, shouldn't `obsolete19` be included on this list?

Reply:
Done and added `obsolete19` to be safe.

Reply:
I suggested `obsolete14` only because that's the one that the unit test uses to test experimental flags.

But I just realized that `obsolete14`, aka `durable_object_alarms`, is actually a flag some people might be using -- it used to be used to enable alarms, but now they are on by default, but someone might still have it in their settings, and so we could inadvertently break them by making it "experimental".

`obsolete19` is a better choice, because the associated flag, `durable_object_rename`, was never actually used by anyone.

So what we should do is:
* Do NOT mark `obsolete14` experimental.
* DO mark `obsolete19` experimental.
* Change the unit test to use `durable_object_rename` instead of `durable_object_alarms` for the purpose of testing the experimental flags.

(Another option would be to create a new flag marked experimental specifically for the test's sake, but I hate to burn a bit in the structure just for a test...)